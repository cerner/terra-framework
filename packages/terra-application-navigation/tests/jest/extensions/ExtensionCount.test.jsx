import React from 'react';
import ExtensionCount from '../../../src/extensions/_ExtensionCount';

describe('ExtensionCount', () => {
  it('should render default element', () => {
    const shallowComponent = shallow(
      <ExtensionCount />,
    );

    expect(shallowComponent).toMatchSnapshot();
  });

  it('should render prop data', () => {
    const shallowComponent = shallow(
      <ExtensionCount
        value={1}
        isRollup
      />,
    );

    expect(shallowComponent).toMatchSnapshot();
  });
});
