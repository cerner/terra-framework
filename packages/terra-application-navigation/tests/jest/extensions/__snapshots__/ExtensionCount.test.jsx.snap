// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExtensionCount should render default element 1`] = `
<div
  className="count"
/>
`;

exports[`ExtensionCount should render prop data 1`] = `
<div
  aria-label={null}
  className="count is-rollup"
/>
`;
